### ISSUE TITLE
Missing some .NET 3.5 assemblies

### ISSUE THREAD

**Issue by WamWooWam** [itsmattkc/dotnet95#15]
Some assemblies from .NET 3.x seem to be missing from the installer.

Example: ShareX is missing `System.Xml.Linq`
![image](https://github.com/itsmattkc/dotnet9x/assets/11337731/52a3d131-2b06-4995-bbe7-99704fa73ffb)

Other assemblies that seem to be missing include 
- `WindowsBase`
- `PresentationFamework.*`
- `PresentationCore` 
- `PresentationUI` 
- `WindowsFormsIntegration`
  - (these all make up WPF which I don't expect to work but some WinForms apps may use them for additional helpers) 
- `System.AddIn`
- `System.ComponentModel.DataAnnotations`
- `System.Data.DataSetExtensions`
- `System.Data.Entity`
- `System.Data.Linq`
- `System.IdentityModel`
- `System.Printing`
- `System.Runtime.Serialization` (v3.0.0.0)
- `System.Speech`
- `System.ServiceModel.*`
- `System.Transactions`
- `System.Workflow.*`
- `System.Net` (v3.5.0.0)

& probably more, my reference system includes Visual Studio and Office 2003 so it's a little hard to parse what's added by those vs. what's included purely in .NET

**Comment by RealCalumPlays** [comment]
Pretty sure all of the libraries .NET installed can be seen in the nsis installer script near the bottom when its adding them to GAC. I may be wrong so take this with a pinch of salt haha.
